andela/hc-beef-sauce
# [WIP 158532841] Add user notify priority work in progress

#### What does this PR do?
Ensure that user priority is defined for every check
#### Description of Task to be completed?
Write the templates and forms for submission
#### How should this be manually tested?
To test it, load the user settings page and click on the priority link in team access table.
A modal form opens up in which the user can assign low, medium and high priority to the check for that user. 
Once submitted(Send edit priority clicked), the user should be able to receive notification on the priority assigned (This is **WIP**)
What are the relevant pivotal tracker stories?
#158532841
#### Screenshots
<img width="1432" alt="screen shot 2018-07-06 at 14 06 01" src="https://user-images.githubusercontent.com/32410518/42376885-7395c6d2-8129-11e8-95ce-ff49c848d201.png">

#### The Current Images on the this updated PR
<img width="1429" alt="screen shot 2018-07-11 at 13 23 45" src="https://user-images.githubusercontent.com/32410518/42566017-d05a815e-850d-11e8-8ac6-b9d9ec5ff06c.png">
<img width="1425" alt="screen shot 2018-07-11 at 13 24 12" src="https://user-images.githubusercontent.com/32410518/42566026-d7f5d5ee-850d-11e8-91e5-7091917a9406.png">
